### DISCUSSION TITLE
Allow multiple connections between operations in a flow

### DISCUSSION THREAD

**Discussion by panda-madness** [directus/directus#17045] [Feature Requests]
Currently, you can only connect operations sequentially. Since each operation adds it's data to the flow data object this is enough to implement a lot of business flows but it forces a somewhat awkward operation schema where potential side-effect operations are mixed inline with the primary operations of the flow. It would be nice to be able to connect the various nodes of a flow arbitrarily, like in other automation software, such as n8n or node-red.

Poorly drawn explanation of what I mean below:

![Screenshot 2023-01-07 at 22 52 21](https://user-images.githubusercontent.com/6180087/211161719-29b5570b-00c2-4235-899c-d6887759af57.png)

**Comment by azrikahar** [comment]
Reminds me of #16356 which is technically the inverse of this, but likely requires similar changes under the hood.

Do feel free to check out #15677 as it is a great discussion to aggregate feedbacks regarding Flows in general 👍

**Comment by azrikahar** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
